docsets/Terraform/docset.json

```diff
@@ -1,13 +1,17 @@
 {
     "name": "Terraform",
-    "version": "0.14.0",
+    "version": "0.14.8",
     "archive": "Terraform.tgz",
```

Comment:
In the future please update the archive at this path as well. I'll do it by copying the archive at `versions/0.14.8/Terraform.tgz`.